sdk/src/replicated-data/ormap.js

```diff
@@ -63,22 +63,6 @@ function ORMap() {
     cleared: false,
   };
 
-  /**
-   * Generator for default values.
-   *
-   * This is invoked by get when the current map has no Replicated Data defined for the key.
-   *
-   * If this returns a Replicated Data object, it will be added to the map.
-   *
-   * Care should be taken when using this, since it means that the get method can trigger elements to be created. If
-   * using default values, the get method should not be used in queries where an empty value for the Replicated Data
-   * means the value is not present.
-   *
-   * @callback module:akkaserverless.replicatedentity.ORMap~defaultValueCallback
-   * @param {module:akkaserverless.Serializable} key The key the default value is being generated for.
-   * @returns {undefined|module:akkaserverless.replicatedentity.ReplicatedData} The default value, or undefined if no default value should be returned.
-   */
```

Comment:
Explanations for how things work are lost now, I think we still need them but in some other form.

Reply:
👍🏼